clustered-indexes.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Clustered Indexes
+summary: Learn how clustered indexes apply to TiDB.
+---
+
+# Clustered Indexes
+
+The clustered index is an experimental feature introduced in TiDB 5.0-rc. This document provides multiple examples to explain how this feature makes a difference to the query performance of TiDB. To enable this feature and see the detailed operation guide, see [tidb_enable_clustered_index](/system-variables.md#tidb_enable_clustered_index).
+
+Clustered indexes provide TiDB the ability to organize tables in a way that can improve the performance of certain queries. The term _clustered_ in this context refers to the _organization of how data is stored_ and not _a group of database servers working together_. Some database management systems refer to clustered indexes as _index-organized tables_ (IOT).
+
+TiDB supports clustering only by a table's `PRIMARY KEY`. With clustered indexes enabled, the terms _the_ `PRIMARY KEY` and _the clustered index_ might be used interchangeably. `PRIMARY KEY` refers to the constraint (a logical property), and clustered index describes the physical implementation of how the data is stored.
```

Comment:
"the terms _ `PRIMARY KEY`_ and _clustered index_ might be used interchangeably. " This is a bit confusing. How can two terms used interchangeably? Could you change the wording?
```suggestion
TiDB supports data clustering only by a table's `PRIMARY KEY`. With clustered indexes enabled, the terms _ `PRIMARY KEY`_ and _clustered index_ might be used interchangeably. `PRIMARY KEY` refers to the constraint (a logical property), and clustered index describes the physical implementation of how the data is stored.
```

Reply:
The sentence that follows explains how they can be used interchangebly. Some databases permit you to chose which index to cluster by, but TiDB does not. Some also permit clustering by moe than one index (such as TokuDB), but TiDB does not. Because TiDB has one clustered index (_the clustered index_) it is always the same index as the `PRIMARY KEY`.

I would prefer without the edits. Data clustering is redundant (clustering already refers to data) and removing "the" makes it less correct for the case with databases that support multiple clustered indexes.